deploy/twingate-operator/values.yaml

```diff
@@ -4,6 +4,9 @@
 
 twingateOperator: {}
```

Comment:
Since apiKey, network, remoteNetworkId are required should we have those uncommented so people don't think they are optional.

Also maybe add a comment to either use apiKey or existingAPIKeySecret.

When not passing either there is an error
```Error: values don't meet the specifications of the schema(s) in the following chart(s):
twingate-operator:
- twingateOperator: Must validate one and only one schema (oneOf)
- twingateOperator: apiKey is required
```
Shoudl that be changed to "apiKey or existingAPIKeySecret1.name is required"

Reply:
I dont control that error information it comes from schema validation...

Reply:
Re uncommenting - I want people to explicitly have to uncomment and put values there rather than run with default values which will seem like its working